core/src/main/java/nl/nn/adapterframework/http/HttpSenderBase.java

```diff
@@ -775,15 +774,24 @@ public String getMethodType() {
 		return methodType.toUpperCase();
 	}
 
-	@IbisDoc({"4", "content-type of the request, only for POST and PUT methods", "text/html"})
+	@Deprecated
+	/**
+	 * This is a superset of mimetype + charset + optional payload metadata.
+	 */
+	@IbisDoc({"4", "superset of mimetype + charset", "automatically determined"})
 	public void setContentType(String string) {
+		setMimeType(string);
```

Comment:
Maar wat voegt het hebben van mimeType nou toe? setContentType doet nu hetzelfde als setMimeType.
Het is ook niet dat het er duidelijker op wordt, toch?
Je moet of een zinnig verschil maken, of één van de twee deprecieren.
Mijn voorkeur heeft: 
- variabele contentType hernoemen naar fullContentType
- variabele mimeType hernoemen naar contentType
- setMimeType() vewijderen